testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/ServerEndpoint.java

```diff
@@ -105,8 +128,9 @@ public void collectSpanAttributes(ServerEndpoint.UrlParameterProvider parameterP
     }
   }
 
-  private static final Map<String, ServerEndpoint> PATH_MAP =
-      Arrays.stream(values()).collect(Collectors.toMap(x -> x.getPath(), x -> x));
+  public static ServerEndpoint[] values() {
```

Comment:
I'm not sure if we will want custom ServerEndpoints in this list, but we can change that behavior if it becomes an issue